processing/src/main/java/io/druid/segment/IndexIO.java

```diff
@@ -1098,6 +1098,11 @@ public static File makeTimeFile(File dir, ByteOrder order)
 
   public static File makeMetricFile(File dir, String metricName, ByteOrder order)
   {
-    return new File(dir, String.format("met_%s_%s.drd", metricName, order));
+    return new File(dir, sanitizeFileName(String.format("met_%s_%s.drd", metricName, order)));
+  }
+
+  public static String sanitizeFileName(String fileName)
+  {
+    return fileName.replace(File.separator, "↗");
```

Comment:
Is replacing separators enough? We should look into something that will work for any special characters.